PyRedPitaya/raw_memory.py

```diff
@@ -1,8 +1,27 @@
+import platform
 from ctypes import POINTER, c_uint32, cast, cdll, create_string_buffer, sizeof
+from os import path
+from time import time
 
 import numpy as np
 
-libmonitor_file = "libmonitor.so"
+if "PyRedPitayaTest" in list(sys.modules.keys()):
+    from PyRedPitayaTest import libmonitor_file
+else:
+    # The path where monitor...so shared object is installed is sort of predictable
+    libmonitor_file = path.join(
+        path.join(
```

Comment:
I think think this should use pathlib instead. It is available since Python 3.4 so should also work on the outdated 3.5 on the RedPitaya.

Reply:
> I think think this should use pathlib instead. It is available since Python 3.4 so should also work on the outdated 3.5 on the RedPitaya.

I won't mind this change, but why?

Reply:
More modern, easier to read.

Reply:
> More modern, easier to read.

Done.